design/MonitoringModeAPI.md

```diff
@@ -83,8 +83,8 @@ see [Update results](/design/UpdateResults.md)
   {
     "version": "1.0",
     "experiment_name": "quarkus-resteasy-autotune-min-http-response-time-db",
-    "start_timestamp": "2022-01-23T18:25:43.511Z",
-    "end_timestamp": "2022-01-23T18:25:43.511Z",
+    "interval_start_time": "2022-01-23T18:25:43.511Z",
+    "interval_end_time": "2022-01-23T18:25:43.511Z",
```

Comment:
Timestamps for interval_start_time and interval_end_time are same, we can change the interval_end_time.